sdf.md

```diff
@@ -587,6 +592,11 @@ affordances, `sdfData`, as well as `minItems` and `maxItems` qualities.
 Therefore, they can be seen as a superset of `sdfObject` groups, additionally
 allowing for composition.
 
+Therefore, an `sdfThing` directly or indirectly contains a set of `sdfProperty`, `sdfAction`, and
+`sdfEvent` definitions that describe the interaction affordances
+associated with some scope of functionality.
```

Comment:
One very minor point: In some places, the source file uses backticks for certain terms `sdfObject`), in other places, no backticks are used. I suppose for the final version, a decision for one of the two approaches needs to be made to arrive at a consistent outcome (which is probably only relevant for the HTML version, right?).

Reply:
That is a good point, and I cannot say we have found the right measure here.
Generally, it is easy to overdo markup such as monospacing. 
So the intent was that `sdfObject` would be used where we describe the syntax of models, and plain sdfObject where we dicuss the concept.
I think it would make sense to do an editorial pass to actually meet this intent, but that can probably be done after this PR is completed.